src/helpers/getDeckAdvice/advice/NORTHSEA_DOG/index.js

```diff
@@ -0,0 +1,34 @@
+export default cards => {
+  const getAverageManaCost = cards =>
+    cards.map(card => card.mana).reduce((a, b) => a + b, 0) / cards.length
+
+  const MANA_THRESHOLD = 3
+  const hasNorthseaDog = cards.map(card => card.id).includes('N5')
+  const hasFirstMutineer = cards.map(card => card.id).includes('N12')
+  const { faction } = cards.find(card => card.faction !== 'neutral') || {}
+  const hasGiftOfTheWise = cards.map(card => card.id).includes('W19')
+  const hasFrozenCore = cards.map(card => card.id).includes('W9')
+  const winterConditions =
+    faction === 'winter' && (hasFrozenCore || hasGiftOfTheWise)
+  const averageManaCost = getAverageManaCost(cards)
+  const winterMessage =
+    faction === 'winter' ? `, or including Gift of the Wise or Frozen Core` : ''
+
+  // For Northsea Dog to be considered efficient, it requires a low mana average
+  // or to include First Mutineer, Gift of the Wise or Frozen Core.
+  if (
+    !hasNorthseaDog ||
+    hasFirstMutineer ||
+    averageManaCost <= MANA_THRESHOLD ||
+    winterConditions
+  )
+    return null
+
+  return {
+    name: 'Inefficient Northsea Dog',
+    description: `This deck includes Northsea Dog but has an average mana cost of ${averageManaCost.toFixed(
```

Comment:
I think the relationship between Northsea Dog and the average mana cost is not quite clear from the first glance. I wonder whether we should make the message a bit more clear.